src/views/FarmAuction/components/AuctionCakeBurn.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react'
+import CountUp from 'react-countup'
+import { Text, Flex, Skeleton } from '@pancakeswap/uikit'
+import { useFarmAuctionContract } from 'hooks/useContract'
+import { useTranslation } from 'contexts/Localization'
+import { usePriceCakeBusd } from 'state/farms/hooks'
+import { getBalanceNumber } from 'utils/formatBalance'
+import { ethersToBigNumber } from 'utils/bigNumber'
+
+const AuctionCakeBurn: React.FC = () => {
+  const [burnedCakeAmount, setBurnedCakeAmount] = useState(null)
+  const { t } = useTranslation()
+  const farmAuctionContract = useFarmAuctionContract()
+  const cakePriceBusd = usePriceCakeBusd()
+
+  const burnedAmountAsUSD = cakePriceBusd.times(burnedCakeAmount).toNumber()
```

Comment:
We should try to stick to the rule of keeping a BN representation until the number is displayed in a component.
Which would involve removing `.toNumber()` here , and replace line 48 by `isNan()` function from the BN lib.

Reply:
Fixed in 6ca621e8b26d77188c952317d2ce3e3b5bcde601